docs/source/developers/ROADMAP_0_3_lookback.md

```diff
@@ -0,0 +1,109 @@
+
+- Support for persistent settings [#834](https://github.com/napari/napari/pull/834) to allow **saving of preferences** between launches of the viewer.
+    
+    > We have not done this work yet, and this work will be moved to our 0.4 series of releases.
+
+
+## Provide accessible documentation, tutorials, and demos
+
+- Improve our website [napari.org](https://napari.org) to provide easy access to all napari related materials, including the [**four types of documentation**](https://www.divio.com/blog/documentation/): learning-oriented tutorials, goal-oriented how-to-guides or galleries, understanding-oriented explanations (including developer resources), and a comprehensive API reference. See [#764](https://github.com/napari/napari/issues/764) and the [`documentation` label](https://github.com/napari/napari/labels/documentation) on the napari repository for more details.
+
+    > We have begun the design work for this but have not begun the implementation yet. This work will be moved to our 0.4 series of releases.
+
+- Add a **napari human interface guide** for plugin developers, akin to [Apple's Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/). We want such a guide to promote best practices and ensure that plugins provide a consistent user experience.
+
+    > We have not done this work yet, and this work will be moved to our 0.4 series of releases.
+
+## Work prioritized for future roadmaps
+
+We’re also planning or working on the following more advanced features, which will likely be prioritized in future roadmaps:
+
+- General support for undo / redo functionality [#474](https://github.com/napari/napari/issues/299), a history feature, and macro generation.
+
+- Complete serialization of the viewer [#851](https://github.com/napari/napari/pull/851) to enable sharing the entire viewer state. This feature will be supported after `writer plugins` have been added.
+
+- Support for generating animations [#780](https://github.com/napari/napari/pull/780). This feature will be supported after we have the ability to serialize the viewer state [#851](https://github.com/napari/napari/pull/851).
+
+- Draggable and resizable layers [#299](https://github.com/napari/napari/issues/299) and [#989](https://github.com/napari/napari/pull/989). This feature will be supported after we have added support for world coordinates [project board 10](https://github.com/napari/napari/projects/10) and rotations to our transform model.
+
+- Linked 1D plots such as histograms, timeseries, or z-profiles [#823](https://github.com/napari/napari/pull/823) and [#675](https://github.com/napari/napari/pull/675).
+
+- Support for using napari with remote computation (i.e. a remote jupyter notebook [#495](https://github.com/napari/napari/issues/495)).
+
+- Functional or interactive plugins that allow for analysis of data or add elements to the GUI.
```

Comment:
Worth writing here that we actually have experimental support for this right now!

Reply:
added line, don't want to say too much as a feel like it belongs more in our 0.4 section

Reply:
wasn't really an 0.3 accomplishment, i might actually not include. it will be highlighted elsewhere